### PR TITLE
REF: bump aar to 106

### PR DESCRIPTION
**Pull request by Overtorment**

### CODE REVIEW

**Comment by Overtorment** [comment]
@TheBlueMatt can you take a look plz?

**Comment by Overtorment** [comment]
@arik-so can you take a look plz?

File: android/src/main/java/com/rnldk/RnLdkModule.kt [hunk 1]
```diff
@@ -262,7 +269,7 @@ class RnLdkModule(private val reactContext: ReactApplicationContext) : ReactCont
           fee_estimator,
           chain_monitor,
           tx_filter,
-          router,
+          router!!.write(),
```
Comment (Overtorment) [inline android/src/main/java/com/rnldk/RnLdkModule.kt:272]: @TheBlueMatt wtf?

Comment (Overtorment) [reply android/src/main/java/com/rnldk/RnLdkModule.kt:272]: especially in below  call its still router, not bytes

File: android/src/main/java/com/rnldk/RnLdkModule.kt [hunk 2]
```diff
@@ -142,6 +142,13 @@ class RnLdkModule(private val reactContext: ReactApplicationContext) : ReactCont
           that.sendEvent(MARKER_PERSIST_MANAGER, params);
         }
       }
+
+      override fun persist_network_graph(network_graph: ByteArray?) {
```
Comment (Overtorment) [inline android/src/main/java/com/rnldk/RnLdkModule.kt:146]: @marcosrdz can you pls check swift part and implement this event handler as well.
@arik-so its weird that it was not forced on swift

Comment (arik-so) [reply android/src/main/java/com/rnldk/RnLdkModule.kt:146]: ok, I'll update the build to make it forced

Comment (arik-so) [reply android/src/main/java/com/rnldk/RnLdkModule.kt:146]: try this: https://github.com/lightningdevkit/ldk-swift/releases/tag/v0.0.106.1

File: android/src/main/java/com/rnldk/RnLdkModule.kt [hunk 3]
```diff
@@ -111,17 +111,17 @@ class RnLdkModule(private val reactContext: ReactApplicationContext) : ReactCont
         val channel_monitor_bytes = data.write()
         println("ReactNativeLDK: persist_new_channel")
         val params = Arguments.createMap()
-        params.putString("id", byteArrayToHex(id.to_channel_id()))
+        params.putString("id", byteArrayToHex(id.write()))
```
Comment (TheBlueMatt) [inline android/src/main/java/com/rnldk/RnLdkModule.kt:114]: So the iOS version is gonna stay `write`, correct?

File: android/src/main/java/com/rnldk/RnLdkModule.kt [hunk 4]
```diff
@@ -142,6 +142,13 @@ class RnLdkModule(private val reactContext: ReactApplicationContext) : ReactCont
           that.sendEvent(MARKER_PERSIST_MANAGER, params);
         }
       }
+
+      override fun persist_network_graph(network_graph: ByteArray?) {
+        println("ReactNativeLDK: persist_network_graph");
+        if (networkGraphPath != "") {
+          File(networkGraphPath).writeBytes(router!!.write());
```
Comment (TheBlueMatt) [inline android/src/main/java/com/rnldk/RnLdkModule.kt:149]: Should just use the bytes provided, no?

File: android/src/main/java/com/rnldk/RnLdkModule.kt (hunk 1 above)
Comment (TheBlueMatt) [reply android/src/main/java/com/rnldk/RnLdkModule.kt:272]: Right, well the intent here is to read it from disk, which should let you drop a bunch of the code around `initialize graph sync`

**Comment by Overtorment** [comment]
@marcosrdz  https://github.com/lightningdevkit/ldk-swift/releases/tag/v0.0.106.2

File: android/src/main/java/com/rnldk/RnLdkModule.kt (hunk 3 above)
Comment (Overtorment) [reply android/src/main/java/com/rnldk/RnLdkModule.kt:114]: yes, in the end i changed android

File: android/src/main/java/com/rnldk/RnLdkModule.kt (hunk 4 above)
Comment (Overtorment) [reply android/src/main/java/com/rnldk/RnLdkModule.kt]: fixed

**Pull request merged**